HedgeModManager/Resources/Codesv2/SonicFrontiers.hmm

```diff
@@ -2446,6 +2538,170 @@ Patch "Disable Thorn Cylinders" in "Gameplay" by "Hyper" does "Disables the cyli
     );
 }
 
+Patch "Enable Flinging off of Terrain in Cyber Space" in "Gameplay" by "WasifBoomz" does
+/*
+Enables the Tumble State in Cyber Space.
+
+Notes;
+- This does not enable being able to trick off of terrain.
+*/
+{
+    long CheckTumbleFuncAddress = ScanSignature
+    (
+        "\x48\x89\x5C\x24\x10\x48\x89\x74\x24\x18\x55\x48\x8B\xEC",
+        "xxxxxxxxxxxxxx"
+    );
+
+    WriteAsmHook
+    (
+        $@"
+            mov  rdx, rax
+            mov  rcx, rbx
+            mov  rax, {(long)CheckTumbleFuncAddress}
+            call rax
+            cmp  byte ptr [rax], 0
+            jnz  end
+            mov  byte ptr [rax], 1
+            mov  dword ptr [rax+4h],  0x41700000
+            mov  dword ptr [rax+1Ch], 0x40000000
+            mov  dword ptr [rax+20h], 0x40000000
+            mov  dword ptr [rax+24h], 0x42480000
+            mov  dword ptr [rax+28h], 0x42B40000
+            mov  dword ptr [rax+2Ch], 0x42200000
+            mov  dword ptr [rax+30h], 0x42A00000
+            mov  dword ptr [rax+34h], 0x41A00000
+            mov  dword ptr [rax+40h], 0x41A00000
+            mov  dword ptr [rax+4Ch], 0x41400000
+        end:
+            cmp  byte ptr [rax], 0
+        ",
+
+        /* 0x1408FD0C5 */
+        ScanSignature
+        (
+            "\x48\x8B\xD0\x48\x8B\xCB\xE8\xCC\xCC\xCC\xCC\x80\x38\x00\x74\x7D",
+            "xxxxxxx????xxxxx"
+        ),
+                
+        HookBehavior.Replace
+    );
+}
+
+Code "Enable Tricks in Cyber Space" in "Gameplay" by "WasifBoomz" does
+/*
+Allows Sonic to use air tricks during Cyber Space levels.
+
+Notes;
+- This does not enable being able to fling off of terrain.
```

Comment:
```
Allows Sonic to use the Air Trick skill in Cyber Space.

Notes;
- This will only work for traversal objects, use the 'Enable Air Tricks from Terrain in Cyber Space' code to allow Sonic to fling off of terrain.
```